pylint_odoo/checkers/no_modules.py

```diff
@@ -521,6 +526,19 @@ def visit_dict(self, node):
                              args=(version_format,
                                    self.config.manifest_version_format_parsed))
 
+        # Check if resource exist
+        data_keys = ['data', 'demo', 'demo_xml', 'init_xml', 'test',
```

Comment:
You should use `_get_manifest_referenced_files` like as `_check_file_not_used` but inverse the sets `-` operation